rpi-eeprom-config

```diff
@@ -97,34 +213,85 @@ class BootloaderImage(object):
 
 def main():
-    parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter, \
-    description='Bootloader EEPROM configuration tool for the Raspberry Pi 4. \
-\n\nThere are 3 operating modes: \
-\n\n1. Output the bootloader configuration stored in an EEPROM image file to \
-the screen (STDOUT): specify only the name of an EEPROM image file using the \
-\'eeprom\' option. \
-\n\n2. Output the bootloader configuration stored in an EEPROM image file to a \
-file: specify the EEPROM image file using the \'eeprom\' option, and the output \
-file using the \'--out\' option.\
-\n\n3. Insert a new bootloader configuration into an EEPROM image file: specify \
-the source EEPROM image file using the \'eeprom\' option and the bootloader \
-configuration file using the \'--config\' option. A new file which is a \
-combination of the EEPROM image file, together with the new bootloader \
-configuration file will be created - specify its name using the \'--out\' option. \
-The new bootloader configuration will replace any configuration present in the \
-source EEPROM image.\
-\n\nBootloader EEPROM images are contained in the \'rpi-eeprom-images\' package,\
- which installs them to the /lib/firmware/raspberrypi/bootloader directory.')
-    parser.add_argument('--config', help='Name of bootloader configuration file')
-    parser.add_argument('--out', help='Name of output file')
-    parser.add_argument('eeprom', help='Name of EEPROM file to use as input')
+    """
+    Utility for reading and writing the configuration file in the
+    Raspberry Pi 4 bootloader EEPROM image.
+    """
+    description = """\
+Bootloader EEPROM configuration tool for the Raspberry Pi 4.
+Operating modes:
+
+1. Output the current bootloader to configuration STDOUT if no arguments are
+   specified OR the given output file if --out is specified.
+
+   rpi-eeprom-config (--out boot.conf)
+
+2. Extracts the configuration file from the given \'eeprom\' file and outputs
+   the result to STDOUT or the output file if --output is specified.
+
+   rpi-eeprom-config pieeprom.bin (--out boot.conf)
+
+3. Writes a new EEPROM image replacing the configuration file with the contents
+   of the file specified by --config.
+
+   rpi-eeprom-config --config boot.conf --out newimage.bin pieeprom.bin
+
+   The new image file can be installed via rpi-eeprom-update
+   rpi-eeprom-update -d -f newimage.bin
+
+4. Applies a given config file to the latest available EEPROM image and
+   invokes rpi-eeprom-update to update the bootloader when the system is
+   rebooted.
+
+   rpi-eeprom-config --apply boot.conf
+
+   The latest available image is determined by querying \'rpi-eeprom-update -l\'
+   and depends on the rpi-eeprom-update configuration.
+
+5. Launches the default text editor ($EDITOR) to edit the current EEPROM
+   configuration and then invokes rpi-eeprom-update to apply the updated
+   configuration to the latest available EEPROM image and schedule an update
```

Comment:
The wording here seems to imply that rpi-eeprom-update **both** "applies the configuration" and "schedules the update", but AFACT it's rpi-eeprom-config itself that "applies the configuration", and rpi-eeprom-update only "schedules the update" ?

Combining it with the wording of Number 4 above, perhaps:
5. Launches the default text editor ($EDITOR) to edit the current EEPROM
   configuration, applies the updated configuration to the latest available
   EEPROM image and then invokes rpi-eeprom-update to update the
   bootloader when the system is rebooted.